src/api/app/views/webui/package/_buildstatus.html.erb

```diff
@@ -5,17 +5,21 @@
       <% previous_repo = nil %>
       <% results.each do |result| %>
         <% repository = @project.repositories.find_by_name(result.repository) %>
-        <% next unless repository %>
-        <% next unless repository.architectures.pluck(:name).include?(result.architecture) %>
-        <tr>
+        <% is_outdated = !(repository && repository.architectures.pluck(:name).include?(result.architecture)) %>
+        <tr class="<%= is_outdated ? 'outdated_buildresult' : '' %>">
+          <td title="<%= result.repository %>" class="no_border_bottom <%= (result.repository == previous_repo) ? " no_border_top" : " "%>">
```

Comment:
The remove of the `rowspan` is needed not because the styling but because before we used `repository.architectures.length` that comes from DB and not from the result set, that leads to a bug in the case of more archs in the DB than in the results (like in the issue #2725 ) and also the other way around.

So, removing the rowspan I don't have to know in advance how many rows I will display, so no out of sync rows...